src/Engine/ProtoCore/Utils/FileUtils.cs

```diff
@@ -23,6 +23,9 @@ public static string GetDSFullPathName(string fileName, Options options = null)
             var trimChars = new[] {'\n','\t','\r',' '};
             fileName = fileName.Trim(trimChars);
 
+            // Fix file paths which include an apostrophe
+            fileName = fileName.Replace("\\'", "'");
```

Comment:
If I understand it correctly this is replacing `\'` with `'`, but how would the test case reflect this fix? Maybe I missed something?